corehq/feature_previews.py

```diff
@@ -127,3 +127,12 @@ def enable_callcenter(domain_name, checked):
     ),
     privilege=LOOKUP_TABLES,
 )
+
+
+SHARE_CASE_TILE_CONTEXT = FeaturePreview(
```

Comment:
i am not completely sure about how feature previews are related to toggles, but it seems alright to move this under `SHOW_PERSIST_CASE_CONTEXT_SETTING` since we are talking about persistent case tiles.